namespaces/cloud-platform-live-0.k8s.integration.dsd.io/support-labelling-app-dev/support-labelling-app-dev-admin-role.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: support-labelling-app-dev-admins 
+  namespace: support-labelling-app-dev # Your namespace `$servicename-$env`
+subjects:
+  - kind: Group
+    name: "github:$cloud-platform-test" # Make this the name of the GitHub team you want to give access to
```

Comment:
Can you update the team name please?
If this is a cloud-platform owned namespace, you could omit the entire file.

Reply:
There is a team called `cloud-platform-test`, I'm using it to test the documentation and workflows for people not in the `cloud-platform` team as `cloud-platform` has elevated privileges.

Reply:
The please remove the `$` :)